contrib/pylightning/lightning-pay

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+
+import argparse
+import random
+import os
+import sys
+from lightning import LightningRpc
+
+parser = argparse.ArgumentParser(formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+parser.add_argument('--bolt11')
+parser.add_argument('amount_in_milli_satoshi', type=int, nargs='?')
+parser.add_argument('destination_id', nargs='?')
+parser.add_argument('payment_hash', nargs='?')
+args = parser.parse_args()
+
+def default_configdir():
+    home = os.getenv("HOME")
+    if home:
+        return os.path.join(home, ".lightning")
+    return "."
+
+rpc_path = os.path.join(default_configdir(), "lightning-rpc")
+l = LightningRpc(rpc_path)
+
+if args.bolt11:
+    bolt11 = l.decodepay(args.bolt11)
+    id_ = bolt11["payee"]
+    payment_hash = bolt11["payment_hash"]
+    if "msatoshi" in bolt11:
+        amount = bolt11["msatoshi"]
+    else:
+        amount = args.amount_in_milli_satoshi
+else:
+    amount = args.amount_in_milli_satoshi
+    id_ = args.destination_id
+    payment_hash = args.payment_hash
+
+route = l.getroute(id_, amount, 1)
+
+fee = route["route"][0]["msatoshi"] - amount
+
+reply = input("Paying fee %s on amount %s (%.3f). Send [Y/n]?" % (fee, amount, fee / amount * 100.0))
```

Comment:
The `%.3f` is a percentage right? Should it not be `%.3f%%` to print the percent symbol also?

Reply:
sure, fixed!